### ISSUE TITLE
No log file written

### ISSUE THREAD

**Issue by fho4train** [POpus/dokuwiki-groupmail#10]
No log written. The plugin has the default logfile name e.g. ":groupmail:log".
Checked the plugin as super user, so permissions should not be a problem.
Created the log file manually as test, but it remains empty.
The sitemap does not show any other file that could be the logfile.
Effect with latest Dokuwiki on Windows10. Other installed plugins:
    Translation Plugin
    CAPTCHA Plugin
    Upgrade Plugin
    Wrap Plugin
    Gallery Plugin
    Move Plugin
    Page Redirect Plugin
    SMTP Plugin

**Comment by POpus** [comment]
Did you ever resolve this? It works on my Ubuntu Linux system.

**Comment by fho4train** [comment]
No, removed the plugin. But I have now also an additional Ubuntu VM. Gives the option to check again and compare. Thanks for the hint. I have currently more work, so it will take some time to dive in this one., but I will respond as soon as I have more info.

**Comment by fho4train** [comment]
Just loaded the groupmail plugin on my laptop and on the Ubuntu VM.
The configuration setting for both installations for the log = ":groupmail:log" (the default)
Logged in as super user and the mail is sent correctly on both installations.
Now I expected in folder data/pages a folder with name "groupmail" and in this folder a file "log" or "log.txt". No folder "groupmail" and no file "log"/"log.txt"

Changing the default ":groupmail:log" to "log" only creates file data/pages/log.txt as expected.
Changing the default ":groupmail:log" to "groupmail:log"creates file data/pages/groupmail/log.txt.
The leading semicolon in the default log seems to be too much.

**Comment by POpus** [comment]
Ok, my results are different.

Changing the default ":groupmail:log" to "log" creates file data/pages/**groupmail**/log.txt.
Changing the default ":groupmail:log" to "groupmail:log" doesn't make a difference. Neither produce a log file.

How are you telling groupmail to send a log to the default directory? Mine looks like this:

{{groupmail>touser=dc10020|subject=Groupmail|content=default directory|sendlog}}

And results in an email like this:

  

  Subject: [_wiki-name_] Groupmail
  
  default directory
  ________________________________________
  
  sent byemtest <xxxxxx@yyyyyy>
  via DokuWiki at http://_wiki-name_/doku.php
  view the message online at  http://_wiki-name_/doku.php?id=groupmail:log
